homeassistant/components/nordpool/strings.json

```diff
@@ -70,9 +70,59 @@
       }
     }
   },
+  "services": {
+    "get_prices_for_date": {
+      "name": "Get prices for date",
+      "description": "Retrieve the prices for a specific date.",
+      "fields": {
+        "config_entry": {
+          "name": "Select Nord Pool configuration entry",
+          "description": "Choose the configuration entry."
+        },
+        "date": {
+          "name": "Date",
+          "description": "Only dates two months in the past and one day in the future is allowed."
+        },
+        "areas": {
+          "name": "Areas",
+          "description": "One or multiple areas to get prices for. If left empty it will use the areas already configured."
+        },
+        "currency": {
+          "name": "Currency",
+          "description": "Currency to get prices in. If left empty it will use the currency already configured."
+        }
+      }
+    }
+  },
+  "selector": {
+    "currency": {
+      "options": {
+        "dkk": "DKK",
+        "eur": "EUR",
+        "nok": "NOK",
+        "pln": "PLN",
+        "sek": "SEK"
+      }
+    }
+  },
```

Comment:
Are these translatable options?

Reply:
I will remove this, no point in translating this (and I think it's not working anyway right now).